apps/explorer/src/ui/StatAmount.tsx

```diff
@@ -21,32 +22,13 @@ export function StatAmount({
     return (
         <div className="flex flex-col justify-start text-sui-grey-75 gap-2">
             <div className="text-sui-grey-100 flex flex-col items-baseline gap-2.5">
-                {date && (
-                    <div className="text-sui-grey-75">
-                        <Text variant="bodySmall" weight="semibold">
-                            {formatDate(date, [
-                                'month',
-                                'day',
-                                'year',
-                                'hour',
-                                'minute',
-                            ])}
-                        </Text>
-                    </div>
-                )}
-                <Heading as="h4" variant="heading4" weight="semibold">
-                    Amount
-                </Heading>
-
-                <div className="flex flex-row items-start gap-1">
-                    <Heading as="h2" variant="heading2" weight="bold">
-                        {amount}
+                {date && <DateCard date={date} />}
+                <div className="flex flex-col items-baseline gap-2.5">
+                    <Heading as="h4" variant="heading4" weight="semibold">
+                        Amount
                     </Heading>
-                    {currency && (
-                        <div className="text-sui-grey-80 ">
-                            <Text variant="bodySmall">{currency}</Text>
-                        </div>
-                    )}
+
+                    <CoinBalance amount={amount} symbol={currency} size="lg" />
```

Comment:
I'm not convinced this component should use coinbalance vs amount, we should try to keep these purely presentation and push the usage of app UI components into the app itself.

Reply:
Correct me if I am wrong, StatAmount is meant to show the transferred amount, which could `Sui`  or another token. The reason I am using the CoinBalance component instead of the Amount Component is  to leverage `useFormatCoin.` The Props for StatAmount are coin-type  `0x2::sui::SUI` amount `1000`.  
<img width="115" alt="Screenshot 2022-11-16 at 7 43 49 AM" src="https://user-images.githubusercontent.com/8676844/202183794-a7b3034e-0611-4638-aac6-a9ade27508ea.png">

Reply:
Got it.